kuiper-game/Systems/SaveLoad.cs

```diff
@@ -14,7 +14,7 @@ public static class SaveLoad
 
         public static void SaveGame(Captain captain) 
         {
-            var captainJson = JsonConvert.SerializeObject(captain);
+            var captainJson = JsonConvert.SerializeObject(captain, new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.Auto});
```

Comment:
Newtonsoft had a hard time with interface types, in this case IShipEvents. Doesn't seem to bloat the save file